money.go

```diff
@@ -107,3 +107,12 @@ func (money *Money) AddTaxPercent(tax int64) *Money {
 func (money *Money) IsZero() bool {
 	return money.Cents() == 0
 }
+
+// Markup adds a percentage of the price to itself with cents precision.
```

Comment:
No son céntimos en los porcentajes... :laughing:  Parte la la línea donde corresponda:

```
 // Markup adds a percentage with decimals of the price to itself. The percentage should be pre-multiplied by 100 to avoid floating point issues.
```